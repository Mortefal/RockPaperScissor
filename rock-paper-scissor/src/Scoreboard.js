import React from 'react';
import Digit from './Digit';

class Scoreboard extends React.Component{
    constructor(props) {
        super(props);
    }

    render() {
        const {user, computer} = this.props;
        return (
            <div>
                <Digit digit={user}/>
                <Digit digit={computer}/>
            </div>
        );
    }

}
export default Scoreboard;