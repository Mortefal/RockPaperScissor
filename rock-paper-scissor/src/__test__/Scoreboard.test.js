import React from "react";
import renderer from 'react-test-renderer';
import Scoreboard from "../Scoreboard";

it('renders correctly', () => {
    const tree = renderer
        .create(<Scoreboard/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});