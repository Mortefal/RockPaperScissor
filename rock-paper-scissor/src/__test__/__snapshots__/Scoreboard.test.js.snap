// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "background": "dimgrey",
      "border": "solid red",
      "height": "75px",
      "justifyContent": "center",
      "margin": "auto",
      "marginTop": "10px",
      "padding": "20px",
      "width": "200px",
    }
  }
>
  <div
    className="row center-xs"
  >
    <div
      className="col-xs-4"
    >
      <div>
        <label
          style={
            Object {
              "fontSize": "48px",
            }
          }
        />
      </div>
    </div>
    <div
      className="col-xs-4"
    >
      <label
        style={
          Object {
            "fontSize": "48px",
          }
        }
      >
        :
      </label>
    </div>
    <div
      className="col-xs-4"
    >
      <div>
        <label
          style={
            Object {
              "fontSize": "48px",
            }
          }
        />
      </div>
    </div>
  </div>
</div>
`;
